pkg/kubelet/pod_workers_test.go

```diff
@@ -33,6 +33,8 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/util/queue"
 )
 
+const numPods = 20
```

Comment:
if you move it up to a constant, it would be good to comment how its used globally.